utils.js

```diff
@@ -13,8 +17,25 @@ function sample(array) {
   return array[rand(0, array.length - 1)]
 }
 
+function readJson(json) {
+  if (fs.existsSync(json)) {
+    if (FILE_INFO_MESSAGES) {
```

Comment:
This could be maybe done with some log level global setting? When set to `DEBUG` this message would be included. See https://sematext.com/blog/logging-levels/ for more info.